test/concurrency/common_checker.h

```diff
@@ -39,6 +100,31 @@ auto Delete(Transaction *txn, BustubInstance &instance, int v1) -> void {
   ASSERT_EQ(ss.str(), "3,\n");
 }
 
+/// @brief Use this to write your own test cases, if needed
+/// @param d_vec Elements to be deleted from t1, represents the value of `t1:v1`
+/// @param d_size The size you expect to be deleted from t1, will be used as sanity check, default value is 3
+/// This overloaded function performs a vectorized self-constructed deletion to better test the implementation
+/// You should calculate the number of elements that logically should be deleted, and pass it in as @d_size.
+auto Delete(Transaction *txn, BustubInstance &instance, const std::vector<int> &d_vec, int d_size = 3) -> void {
+  if (d_vec.empty()) {
+    fmt::print(stderr, "Input vec must not be empty\n");
+    return;
+  }
+
+  assert(!d_vec.empty());
+
+  std::stringstream ss;
+  auto writer = bustub::SimpleStreamWriter(ss, true, ",");
+
+  for (const auto &v1 : d_vec) {
+    fmt::print(stderr, "delete data with v1 = {} in txn {} {}\n", v1, txn->GetTransactionId(), txn->GetIsolationLevel());
+    std::string sql = fmt::format("DELETE FROM t1 WHERE v1 = {}", v1);
+    instance.ExecuteSqlTxn(sql, writer, txn);
```

Comment:
check return value == true otherwise print a warning

Reply:
Sure, I also changed the sequence of printing messages (&& add result checker) for `Insert()`